### DISCUSSION TITLE
Export a particular **frame** as `.png`

### DISCUSSION THREAD

**Discussion by nveshaan** [ManimCommunity/manim#2812] [Suggestions and Proposals]
## Description of proposed feature

A new CLI flag to save a particular **frame** as `.png`. This flag is similar to `--save_last_frame`, but this can save any frame, not just the last.

## How can the new feature be used?

Users can use this flag to save a snapshot of an animation that is still in progress.

## Additional comments

If a **frame** sounds a bit difficult to use, **seconds** can be used instead.

**Comment by behackl** [comment]
How would one specify the particular frame that is desired? The last frame is easy to identify, but how would a user specify a frame in the middle of some animation?

**Comment by nveshaan** [comment]
users have to do some math here, 
for example, an animation of creation of a circle(with rate_func set to linear) is 2 sec long with 60 fps, and the user wants a snapshot when the circle is half-drawn(a semi-circular arc).
he can either give the 60th frame as the argument(since the animation is 120 frames, the circle is half-drawn at 60th frame) or 1 sec as the argument(since the animation is 2 sec long and the circle is half-drawn after half of the animation time is passed)

P.S. : sorry for the clumsy comment, little busy here

**Comment by nveshaan** [comment]
users may also give an argument like the (% of the animation completed) when the snapshot needs to be taken
